src/it/resources/org/checkstyle/suppressionxpathfilter/equalshashcode/SuppressionXpathRegressionEqualsHashCodeNestedCase.java

```diff
@@ -0,0 +1,16 @@
+package org.checkstyle.suppressionxpathfilter.equalshashcode;
+
+public class SuppressionXpathRegressionEqualsHashCodeNestedCase {
+    public boolean equals(Object obj) { // warn
```

Comment:
This example is same as SuppressionXpathRegressionEqualsHashCode1
.

Please remove equals here and remove hashCode in innerClass.
This will make warning on inner class, so xpath will be very different

Reply:
Updated as requested